gossip/src/crds.rs

```diff
@@ -62,7 +62,7 @@ const VOTE_SLOTS_METRICS_CAP: usize = 100;
 //      mainnet: ~280k
```

Comment:
might be nice to refresh these numbers and add a date collected if we have the data in hand.
(if we don't, forget it, they're still close enough-ish)